db/setup/schema.sql

```diff
@@ -27,6 +27,14 @@ CREATE TABLE
   IF NOT EXISTS AuthProvider (
     "providerId" TEXT NOT NULL,
     "provider" ProviderType NOT NULL,
-    "userId" UUID REFERENCES Users(id),
+    "userId" UUID REFERENCES Users (id) ON DELETE CASCADE,
```

Comment:
what does on delete cascade do?

> ON DELETE CASCADE clause in MySQL is used to automatically remove the matching records from the child table when we delete the rows from the parent table. It is a kind of referential action related to the foreign key.

Which is here the child table? Can you make an example, please?

Reply:
If I am not wrong, the child table is this one, the one with `ON DELETE CASCADE`. So, for example, if you delete a user, his auths, blocks, ... (games, friends,... when we have everything ready) would be deleted automatically "on cascade".